posthog/temporal/data_modeling/run_workflow.py

```diff
@@ -997,8 +1002,9 @@ def __init__(self, invalid_input: str):
 @temporalio.activity.defn
 async def build_dag_activity(inputs: BuildDagActivityInputs) -> DAG:
     """Construct a DAG from provided selector inputs."""
+    bind_contextvars(team_id=inputs.team_id)
+    logger = LOGGER.bind()
 
-    logger = await bind_temporal_worker_logger(inputs.team_id)
     await logger.adebug(f"starting build_dag_activity. selectors = {[select.label for select in inputs.select]}")
```

Comment:
From the readme, it sounds like it's now better to do synchronous log calls, is that right? (I can change all of these after release if so)

Reply:
In this PR I didn't want to change anything in the current behavior though. So I've kept all the async usage.